beacon-chain/node/node.go

```diff
@@ -549,6 +555,53 @@ func (b *BeaconNode) registerPOWChainService() error {
 	return b.services.RegisterService(web3Service)
 }
 
+func httpEndpoint(flag string) powchain.HttpEndpoint {
```

Comment:
this really feels like the wrong place to be parsing it for `auth` . This should be handled in the powchain package.